requirements-dev.txt

```diff
@@ -10,3 +10,12 @@ sphinxcontrib-programoutput==0.17
 pytest-asyncio==0.12.0  # pinned to an older version due to an incompatibility with flaky
 greenlet  # used by a test
 nox
+numpy # used by an example
+pre-commit # used to run pre-commit hooks
+# ----------------------------------------------------------
+# packages useful for diagnosing & fixing CI check failures
+#------------------------------------------------------------
+isort==5.8.0
+black==20.8b1
+codespell==2.1.0
+pyright # for local debugging only; not for production use in CI
```

Comment:
It probably makes sense just to use these through pre-commit, especially pyright, which pre-commit can install from the official npm package rather than this 3rd party one.

```suggestion
```

Reply:
Sounds good to me; removed in [`235795ad`](https://github.com/joerick/pyinstrument/pull/161/commits/235795ad4551635819ad04719b19b6160dcc050f).